### PR TITLE
Consistency check

### PR DESCRIPTION
**Pull request by vrothberg**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -91,6 +91,12 @@ jobs:
           command: |
             make vendor
             hack/tree_status.sh
+      - run:
+          name: check profile consistency
+          command: |
+            make default.json
```
Comment (saschagrunert) [inline .circleci/config.yml]: Do we want to adapt the make target to directly overwrite `seccomp.json`. To me it was a bit confusing having both laying (locally) around within the repo.

Comment (vrothberg) [reply .circleci/config.yml]: Nice idea, I'll add that to this PR.

**Comment by vrothberg** [comment]
@saschagrunert PTanotherL

File: Makefile [hunk 2]
```diff
@@ -7,12 +7,12 @@ PACKAGE := github.com/seccomp/containers-golang
 
 sources := $(wildcard *.go)
 
-default.json: $(sources)
+seccomp.json: $(sources)
```
Comment (saschagrunert) [inline Makefile:11]: I think we have to make this `.PHONY`, right?

Comment (vrothberg) [reply Makefile:11]: Good catch! Updated.

**Review by saschagrunert — APPROVED** [review]
LGTM

**Comment by vrothberg** [comment]
@rhatdan PTAL

**Comment by vrothberg** [comment]
(rebased on master now)

**Comment by saschagrunert** [comment]
Let's tag 0.3.1 after this one :)

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**